src/components/MapLayers.js

```diff
@@ -4,14 +4,16 @@ import { Source, Layer } from 'react-mapbox-gl';
 
 class MapLayers extends Component {
     render() {
-        let { landDataLayers } = this.props;
+        const { landDataLayers } = this.props;
+
+        console.log(landDataLayers)
```

Comment:
Sorry this is quite pedantic, but please could we add a message to this log to indicate what it is, or remove the log if it's not needed anymore? There is lots of data that gets logged to the console without any context, which makes it more difficult to debug elsewhere.